python/ray/util/client/common.py

```diff
@@ -215,15 +215,6 @@ def __init__(self,
                     inspect.getmembers(actor_class.actor_cls,
                                        is_function_or_method)).keys())
 
-    def __del__(self) -> None:
-        if ray is None:
-            # The ray API stub might be set to None when the script exits.
-            # Should be safe to skip call_release in this case, since the
-            # client should have already disconnected at this point.
-            return
-        if ray.is_connected():
-            ray.call_release(self.actor_ref.id)
```

Comment:
Is this reference counting happening somewhere else?

Reply:
Yes, it is already handled by https://github.com/ray-project/ray/blob/e3fb9650b2337d9277f757aac797eba1d4d308fb/python/ray/includes/unique_ids.pxi#L324.

Reply:
Perfect!

Reply:
@ckw017 added this pretty recently. I wondered if this change was recently.

Reply:
I think `ray.call_release()` has always been there.

Reply:
if it was always there, then why we needed this @ckw017 ?

Reply:
The change that added this logic is from 9 months ago: https://github.com/ray-project/ray/blame/master/python/ray/util/client/common.py#L224. We probably thought we need this at that time.